public/index.html

```diff
@@ -26,6 +27,7 @@
       name="description"
       content="Ambianic UI progressive web application to manage Ambianic Edge devices"
     />
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
```

Comment:
@vickywane can we pull this icon lib as a local install dependency for improved response time? I've noticed remote dependencies tend to break lighthouse if the serving host has low response time. Pulling as a local dependency makes them part of the CDN propagation. Some libraries are already hosted on CDNs and its OK to link to the their CDN url. 

Looks like iconify has its own public CDN: https://docs.iconify.design/sources/api/hosting.html

However I don't find icnonify in popular js cdn networks: https://cdnjs.com/libraries?q=iconify

What are your thoughts?

Reply:
I think we can make do with now. The iconify CDN is quite widely used by a lot of projects and should have a good response time. It's also cached by the browser so you don't need to pull it each time. 

I tested the display without an internet connection

Reply:
OK